plugin-gradle/src/main/java/com/diffplug/gradle/spotless/KotlinExtension.java

```diff
@@ -122,6 +124,38 @@ private FormatterStep createStep() {
 		}
 	}
 
+	/** Adds the specified version of [diktat](https://github.com/cqfn/diKTat). */
+	public DiktatFormatExtension diktat(String version) {
+		Objects.requireNonNull(version);
+		return new DiktatFormatExtension(version, null);
+	}
+
+	public DiktatFormatExtension diktat() {
+		return diktat(DiktatStep.defaultVersionDiktat());
+	}
+
+	public class DiktatFormatExtension {
+
+		private final String version;
+		private String configPath;
+
+		DiktatFormatExtension(String version, String configPath) {
+			this.version = version;
+			this.configPath =  configPath;
```

Comment:
I wonder if spotless runs some checks on own code, it seems like there should be one space instead of two. Could you please take a look at their CI configs? Or simply run `gradle check`.